src/apis/impl/AuthenticationApiImpl.ts

```diff
@@ -206,7 +223,7 @@ export class AuthenticationApiImpl implements AuthenticationApi {
   async initUserCrypto(api: MedTechApi, token: string, user: User, patientKeyPair: [string, string]): Promise<MedTechApi> {
     const dataOwnerId = user.healthcarePartyId ?? user.patientId ?? user.deviceId;
     if (!dataOwnerId) {
-      throw Error("Invalid user, no data owner id")
+      throw this.errorHandler.createErrorWithMessage("Invalid user, no data owner id")
```

Comment:
`User with id ${loggedUser.id} is not a Data Owner. To be a Data Owner, your user needs to have either patientId, healthcarePartyId or deviceId filled`